watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -1259,20 +1248,19 @@ def _molar_volume_phase_comp(self):
         )
 
     def _diffus_phase_comp(self):
+        # Retrieve component string names from diffusivity_data configuration
+        diffus_data_indices = {i[1] for i in self.params.config.diffusivity_data.keys()}
+        # Retrieve component string names from molar_volume_data configuration
+        molar_volume_data_indices = {
+            i[1] for i in self.params.config.molar_volume_data.keys()
+        }
+        missing_diffus_ind = [
+            i
+            for i in self.params.solute_set
+            if i not in (molar_volume_data_indices | diffus_data_indices)
+        ]
+
```

Comment:
I don't really get why to move these indices outside the method because they were supposed to be useful only under that specific H-L method.

Reply:
Moving the indices are actually consolidating them. If you review the original code, find that the indices are constructed on line 403 (when `self.config.diffus_calculation == DiffusivityCalculation.none`) to log the warning and line 1263 (when `self.config.diffus_calculation == DiffusivityCalculation.HaydukLaudie`) to build the actual H-L calculation.